src/templates/React InstantSearch widget/src/lib/__tests__/dummy-test.tsx.hbs

```diff
@@ -0,0 +1,40 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+// @ts-expect-error
```

Comment:
Can we avoid this by declaring an empty module for all React InstantSearch modules?

Reply:
We already have [this](https://github.com/algolia/create-instantsearch-app/pull/456/files#diff-7e1d0dcc2d8cee7290cd308ce9c29ad0ec82b1a66c54d38e06650f76cd41e45fR47-R49). I googled but couldn't find a way to allow arbitrary members except for what I specified.